#goal
uint32 navigation_mode
uint32 initial_index
float64 goal_latitude
float64 goal_longitude
---
#result
---
#feedback
uint32 current_index
float64 current_latitude
float64 current_longitude
